README.md

```diff
@@ -72,7 +60,6 @@ Each render is passed through a set of HOC's (Higher Order Components) to enhanc
 - withVariant
 - withContexts
 - withPageWrapper
-- withLoading
```

Comment:
Was withLoading a react-router thing, or is it something that we no longer need/use, therefore 🗑️ ?

Reply:
It was a custom higher-order component that appears to only be used during clientside rendering:
```
const LoadingContainer = ({ loading, ...props }) => {
    const [showLoading, setShowLoading] = useState(false);
    useEffect(() => {
      if (loading) {
        timeout = setTimeout(() => {
          setShowLoading(true);
        }, 500);
      }
      return () => clearTimeout(timeout);
    }, [loading]);

    if (!loading) return <Component {...props} />;

    return (
      <LoadingMain role="main">
        <GridWrapper>
          <GridItemMedium>
            {showLoading && <div data-testid="loading" />}
          </GridItemMedium>
        </GridWrapper>
      </LoadingMain>
    );
```

Looks like it implements a fake 500ms loader, which I guess would have been used during page transitions in client-side routing.

Reply:
Thanks for explaining. Happy for it to be deleted!!